shepherd/errors/api.py

```diff
@@ -1,30 +1,25 @@
-__all__ = ['AppError', 'ClientActionError', 'UnknownSheepError', 'UnknownJobError', 'StorageError',
-           'StorageInaccessibleError', 'NameConflictError']
-
+from apistrap.errors import ApiClientError, ApiServerError
 
-class AppError(Exception, BaseException):
-    """Base application exception, will result in 500 response."""
 
-
-class ClientActionError(AppError):
-    """Base client-side error exception, will result in 400 response."""
+__all__ = ['ApiServerError', 'ApiClientError', 'UnknownSheepError', 'UnknownJobError', 'StorageError',
```

Comment:
I intentionally expose `ApiServerError` and `ApiClientError` from `apistrap.errors`.

I think it's better to do this
```
from shepherd.errors.api import ApiClientError, NameConflictError, ..
```

than 
```
from apistrap.errors import ApiClientError
from shepherd.errors.api import NameConflictError, ..
```

However, it feels somehow odd. What do you think, @Teyras ?

Reply:
Eh, whatever.